pkg/shell/hosts_dialog.jsx

```diff
@@ -779,27 +781,32 @@ class ChangeAuth extends React.Component {
                 auto_text = _("Create a new SSH key and authorize it");
                 auto_details = <>
                     <p>{cockpit.format(_("A new SSH key at $0 will be created for $1 on $2 and it will be added to the $3 file of $4 on $5."), key, luser, lhost, afile, ruser, rhost)}</p>
-                    <FormGroup label={_("Key password")} validated={this.state.login_setup_new_key_password_error ? "error" : "default"} helperTextInvalid={this.state.login_setup_new_key_password_error}>
+                    <FormGroup label={_("Key password")}>
                         <TextInput id="login-setup-new-key-password" onChange={value => this.setState({ login_setup_new_key_password: value })}
                                 type="password" value={this.state.login_setup_new_key_password} validated={this.state.login_setup_new_key_password_error ? "error" : "default"} />
+                        <FormHelper helperTextInvalid={this.state.login_setup_new_key_password_error} />
```

Comment:
Where did the validation condition go? (Same for the other 3 instances below)

Reply:
helperTextInvalid={this.state.login_setup_new_key_password_error} 

And then the shared component sets the variant because helperTextInvalid is set https://github.com/KKoukiou/cockpit/blob/helper-text-pf5-compatible/pkg/lib/cockpit-components-form-helper.jsx#L26.

So for the shared component I changed the behavior compared to FOrmGroup to avoid having to pass so many properties. In specific if helperTextInvalid is passed, then it will be shown and the variant set to error.

Reply:
Thanks!